schema/deploy/warehouse/site/data.sql

```diff
@@ -5,44 +5,113 @@ begin;
 
 insert into warehouse.site(identifier, details)
     values
+        ('Apple',                                   '{"category": "community",  "type": "apple-study", 
+            "sample_origin": "apple"}'),
+        ('AvalonFederalWayCareCenter',              '{"category": "community",  "type": "shelter", 
```

Comment:
I saw this site mentioned in lists of skilled nursing facilities (e.g. https://seattle-flu-study.slack.com/archives/C0121DE5FEH/p1588968986325500). Does it also operate as a shelter or should the type be switched?

Reply:
@kschwabefry you are correct, avalon should be a snf, not a shelter